extensions/machine-learning/src/views/models/tableSelectionComponent.ts

```diff
@@ -209,7 +213,7 @@ export class TableSelectionComponent extends ModelViewBase implements IDataCompo
 	}
 
 	public get isDataValid(): boolean {
-		return this.data !== undefined && this.data.databaseName !== this._settings.defaultDbName && this.data.tableName !== this._settings.defaultTableName;
+		return this.data !== undefined && this.data.databaseName !== '' && this.data.tableName !== '' && this.data.databaseName !== undefined && this.data.tableName !== undefined && this.data.databaseName !== this._settings.defaultDbName && this.data.tableName !== this._settings.defaultTableName;
```

Comment:
can we also make sure that the empty square box isnt visible when user just lands on the page (Issue 1 in the bug)

Reply:
that's similar to current design. 
<img width="645" alt="Screen Shot 2020-11-05 at 2 52 27 PM" src="https://user-images.githubusercontent.com/17187348/98305438-b6e51300-1f76-11eb-977a-0e6bc69edd38.png">
maybe we should revisit?

Reply:
this box looks bit more grey and has 'leran more' but the actual box doesnt but I agree with you that lets not remove it for this release if this is as per UX mockups and we should revisit post this release. This box totally confused me - doesnt seem like we need it atleast till we have 'learn more'. lets discuss next week.

@MsSQLGirl  FYI